Controllers/auth/verifyUser.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const jwt = require('jsonwebtoken');
+
+const verifyAccessToken = (req, res, next) => {
+  if (req.headers.access_token) {
+    const pubKey = fs.readFileSync('../../rsa_secret.pub');
+    jwt.verify(req.headers.access_token, pubKey, (error, decoded) => {
+      if (error) {
+        res.status(200).json({
```

Comment:
I guess you can respond with some 4xx status here... As it is unauthorized access.

Reply:
sure